### PR TITLE
Smoother startup

### LINKED ISSUE #1933: Remove splash screen
It seams like Electrons startup is now faster than it used to be. Now the DOM takes longer to load than the splash screen itself. What do you think, should we remove it entirely and focus our efforts on providing a smoother initial startup?

This is 0.3.1 (the splash screen doesn't even appear):
kapture 2017-10-04 at 12 43 56

### PR DESCRIPTION
**Pull request by lgeiger**
- Only show the window once it's ready
- Close the splash screen once 

Closes #1933

### CODE REVIEW

File: packages/desktop/src/notebook/index.js [hunk 1]
```diff
@@ -48,6 +49,7 @@ class App extends React.PureComponent<Object, Object> {
 
   componentDidMount(): void {
     store.dispatch(setNotificationSystem(this.notificationSystem));
+    ipc.send("react-ready");
```
Comment (rgbkrk) [inline packages/desktop/src/notebook/index.js:52]: What a superb idea.

File: packages/desktop/static/index.html [hunk 2]
```diff
@@ -6,47 +6,8 @@
...
-          }
-          el.appendChild(headerEl);
-          el.appendChild(msgContainer);
-          console.dir(err);
-        }
-      };
-    </script>
+    <div id="app"></div>
+    <script src="../lib/vendor.js"></script>
+    <script src="../lib/webpacked-notebook.js"></script>
```
Comment (rgbkrk) [inline packages/desktop/static/index.html:11]: I've got a funny feeling about this not being a require statement, though I can't recall why...

Comment (rgbkrk) [reply packages/desktop/static/index.html:11]: Maybe it was pre-webpack days that this not being `require`'d was odd.

**Comment by rgbkrk** [comment]
With all these sweet changes you're putting in so quickly along with autoupdate, I'm definitely ready to start shipping regularly.

**Comment by rgbkrk** [comment]
smooth

**Comment by lgeiger** [comment]
Looking great. The release build will be even faster ;)

I wonder if we should move back to the static splash screen since the video still takes quite a bit to initialize.

**Comment by lgeiger** [comment]
Also it would be great to initialize the browser window with the correct background color to get rid of the flickering with dark themes.

Do you know if we really need to wait for the `did-finish-load` and `will-navigate` events before we can load the notebook?

**Comment by rgbkrk** [comment]
> I wonder if we should move back to the static splash screen since the video still takes quite a bit to initialize.

I'm ok with the video for the moment. I think it's the electron app readiness that takes the longest here, not the video.

**Comment by rgbkrk** [comment]
> Also it would be great to initialize the browser window with the correct background color to get rid of the flickering with dark themes.

That would be nice. We should probably do some "templating" of our `index.html` for this, unless you're saying a setting on the `BrowserWindow`.

**Comment by rgbkrk** [comment]
> Do you know if we really need to wait for the `did-finish-load` and `will-navigate` events before we can load the notebook?

Not sure.

**Comment by lgeiger** [comment]
> We should probably do some "templating" of our index.html for this, unless you're saying a setting on the BrowserWindow.

Yes it can be configured in the [BrowserWindow](https://electron.atom.io/docs/api/browser-window/#new-browserwindowoptions).

**Pull request merged**